src/sentry/grouping/strategies/configurations.py

```diff
@@ -158,8 +158,7 @@ def register_strategy_config(id: str, **kwargs) -> type[StrategyConfiguration]:
 )
 
 register_strategy_config(
-    id="mobile:2021-02-12",
-    base="newstyle:2019-10-29",
+    id="newstyle_hierarchical:2021-02-12",
```

Comment:
Changing the prefix from `mobile` to `newstyle_hierarchical` and having no base loads `newstyle_hierarchical.py`

This can easily be tested if you remove `newstyle_hierarchical.py` and try to run the tests.

This also requires moving the snapshots:
<img width="642" alt="image" src="https://github.com/user-attachments/assets/1f40d8bf-8785-41fe-9e75-7dba0b414f9a">